torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7754,12 +7845,21 @@ def reference_mse_loss(input, target, reduction="mean"):
         return se
 
 
-def wrapper_set_seed(op, input, *args, **kwargs):
+def wrapper_set_seed(op, *args, **kwargs):
     """Wrapper to set seed manually for some functions like dropout
     See: https://github.com/pytorch/pytorch/pull/62315#issuecomment-896143189 for more details.
     """
+    # freeze_rng_state causes test_composite_compliance to fail.
+    # Once the issure is resolved, think about replacing the code below with
+    # from torch.testing._internal.common_utils import freeze_rng_state
+    # with freeze_rng_state():
+    #     torch.manual_seed(42)
+    #     return op(*args, **kwargs)
+    # This code could be future-proof against accidental
+    # changes in the global random state.
+
     torch.manual_seed(42)
-    return op(input, *args, **kwargs)
+    return op(*args, **kwargs)
```

Comment:
@mruberry You'll bump into this when reviewing this, but I'm interested. What are the implications of this?

Reply:
I do not expect any issues as of now, given that seed is reset prior to each test, and all the samples are generated before calls to ops. This, however, might changed if and when OpInfo test will deal with generators and the processing pipeline changes to `sample1 -> op(sample1) -> sample2 -> op(sample2) -> ...`.